pynest/nest/lib/hl_api_connections.py

```diff
@@ -457,23 +436,8 @@ def CGConnect(pre, post, cg, parameter_map=None, model="static_synapse"):
     if parameter_map is None:
         parameter_map = {}
 
-    if _is_subnet_instance(pre[:1]):
-
-        if not _is_subnet_instance(post[:1]):
-            raise kernel.NESTError(
-                "if pre is a subnet, post also has to be a subnet")
-
-        if len(pre) > 1 or len(post) > 1:
-            raise kernel.NESTError(
-                "the length of pre and post has to be 1 if subnets " +
-                "are given")
-
-        sli_func('CGConnect', cg, pre[0], post[0],
-                 parameter_map, '/'+model, litconv=True)
-
-    else:
-        sli_func('CGConnect', cg, pre, post,
-                 parameter_map, '/'+model, litconv=True)
+    sli_func('CGConnect', cg, pre, post,
+             parameter_map, '/'+model, litconv=True)
 
 
 @check_stack
```

Comment:
Applies to line 445 below: typo "correcponding" -> "corresponding".

Applies to lines 459 and 490 below: I am confused by the word "Description" on a single line in the docstrings.

Reply:
Typo fixed in fbd843400, stray word "Description" removed in bd2fdfe62.